### ISSUE TITLE
[Quest] [Chaos] Ch. 22 quest "Rewards for the Faithful" cannot be completed (missing NPC?)

### ISSUE THREAD

**Issue by lennip** [WarEmu/WarBugs#11692]
Chaos quest Rewards for the Faithful at chapter 22 cannot be completed.
![mutty_003](https://user-images.githubusercontent.com/2891053/39409429-273d9778-4be7-11e8-85f0-2196b655c7fc.jpg)

Problem is the last objective is to kill "Reikland Militia" and after searching the area extensively, no NPC correspond to that name. There are "Reikland Militiaman" in the area, but killing them does not count. I also tried killing every other type of "Reikland whatever" NPCs I could find, in case the name was wrong, but none of them triggers the quest objective counter.

Also, the quest marker is missing:
![mutty_004](https://user-images.githubusercontent.com/2891053/39409465-e5a61b9a-4be7-11e8-8d5f-ada5cf730efa.jpg)

**Comment by lennip** [comment]
UPDATE: I just found where some of the "Reikland Militia" are, completely by chance: some of these NPCs actually appear near the "Ambush at Garrisonburg" PQ at Stage 2. 

![mutty_024](https://user-images.githubusercontent.com/2891053/39469848-9fa9aa36-4d3a-11e8-9e6a-447c438c1d3d.jpg)

However I don't think they should be there, since it is a PQ near chapter 20 (northern part of the map) and very far from chapter 22 area (south). Also they only spawn at Stage II of the PQ, they are not normally there...

To sum it up: Reikland Militia NPCs are not completely missing but they are not where they should be.

**Comment by sioding** [comment]
Thanks for the report. I have created the questmarkers. some npc have been moved to bring the quest in the right direction. fixed with next patch

**Comment by sioding** [comment]
fixed with patch from 05.05.2018
